hikari/impl/interaction_server.py

```diff
@@ -282,7 +291,6 @@ async def aiohttp_hook(self, request: aiohttp.web.Request) -> aiohttp.web.Respon
                 status=_UNSUPPORTED_MEDIA_TYPE_STATUS,
                 body=b"Unsupported Media Type",
                 content_type=_TEXT_CONTENT_TYPE,
-                charset=_UTF_8_CHARSET,
```

Comment:
These are necessary, because they are passed directly to `aiohttp.web.Response`. I was refering to the ones used in `_Response`

Reply:
Yes I saw the mistake later